open-craft/edx-platform
# [SE-5135] Update HTML templates to the v2 counterparts.

## Description

For Studio users updates templates to the v2 counterparts.

![Screenshot_20220121_115125](https://user-images.githubusercontent.com/1470652/150505841-0dec8f34-e803-48be-87a1-b12d2a3710a0.png)
![Screenshot_20220121_115148](https://user-images.githubusercontent.com/1470652/150505827-75267969-6096-4a3e-993b-670f539095ae.png)

## Supporting information

- https://tasks.opencraft.com/browse/SE-5135

## Testing instructions

- While editing a section in the studio, click on the HTML block.
- You will see the list of templates in this commit. Eg. Calendar.

## Deadline

21 Jan 2022.
